ivy_tests/test_ivy/test_functional/test_core/test_creation.py

```diff
@@ -58,13 +59,14 @@ def test_native_array(
 @given(
     dtype_and_start_stop=helpers.dtype_and_values(
         available_dtypes=ivy_np.valid_numeric_dtypes,
-        num_arrays=1,
-        allow_inf=False,
+        num_arrays=2,
+        min_value=None,
+        max_value=None,
         min_num_dims=1,
         max_num_dims=1,
         min_dim_size=2,
         max_dim_size=2,
-        safety_factor=0.5,
+        shared_dtype=True,
```

Comment:
I think you might need to keep this safey factor? I observed some overflow and underflow errors without it.

Reply:
Yea should have been kept in